File: py/P62.py
m = {}
i = 1

while True:
    i += 1
    c = i * i * i
    l = list(str(c))
    l.sort()
    k = "".join(l)
    if k in m:
        l = m[k]
        l.append(c)
        if len(l) == 5:
            for x in l:
                print(f"k: {k}, c: {x}")
            break
    else:
        m[k] = [c]

File: js/P62.js
function print(s){
  process.stdout.write(s + "\n");
}

var m = {};
var i = 1;

while(true){
  i += 1;
  c = i * i * i;
  k = c.toString().split("").sort().join("");
  m[k] = (m[k] == undefined
          ? [c]
          : m[k].concat([c]));

  if(m[k].length == 5){
    m[k].forEach(a => print(a));
    break;
  }
}
